### PR TITLE
[tests-only] e2e tests for url stability open file directly in the web

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This PR adds tests for opening a file/folder directly on the web. In case the web has text editor installed the file will directly open in the text editor and that's what's covered in the test here

## Related Issue

- Part of: https://github.com/owncloud/web/issues/9897

## How Has This Been Tested?

- locally
- ci

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] forward port to master

### CODE REVIEW

File: tests/e2e/cucumber/features/smoke/urlJourneys.feature [hunk 1]
```diff
@@ -24,3 +27,15 @@ Feature: web can be navigated through urls
     And "Alice" shares the following resource using the direct url navigation
       | resource  | recipient | type | role     | resourceType |
       | lorem.txt | Brian     | user | Can view | file         |
+    When "Alice" opens the file "lorem.txt" directly in the browser
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/urlJourneys.feature:30]: let's make it similar to other url navigation steps
```suggestion
    When "Alice" opens the file "lorem.txt" through the URL
```

File: tests/e2e/support/objects/url-navigation/actions.ts [hunk 2]
```diff
@@ -10,6 +10,12 @@ export interface navigateToDetailsPanelOfResourceArgs {
   user: User
 }
 
+export interface openResourceDirectlyInTheBrowserArgs {
```
Comment (saw-jan) [inline tests/e2e/support/objects/url-navigation/actions.ts:13]:
```suggestion
export interface openResourceViaUrlArgs {
```

File: tests/e2e/support/objects/url-navigation/actions.ts [hunk 3]
```diff
@@ -23,3 +29,17 @@ export const navigateToDetailsPanelOfResource = async (
   const fullUrl = `${config.backendUrl}/f/${fileId}?details=${detailsPanel}`
   await page.goto(fullUrl)
 }
+
+export const openResourceDirectlyInTheBrowser = async (
```
Comment (saw-jan) [inline tests/e2e/support/objects/url-navigation/actions.ts:33]:
```suggestion
export const openResourceViaUrl = async (
```

File: tests/e2e/support/objects/url-navigation/index.ts [hunk 4]
```diff
@@ -13,4 +13,10 @@ export class URLNavigation {
   ): Promise<void> {
     await po.navigateToDetailsPanelOfResource({ ...args, page: this.#page })
   }
+
+  async openResourceDirectlyInTheBrowser(
```
Comment (saw-jan) [inline tests/e2e/support/objects/url-navigation/index.ts:17]:
```suggestion
  async openResourceViaUrl(
```

**Review by saw-jan — APPROVED** [review]
:+1:

File: tests/e2e/cucumber/features/smoke/urlJourneys.feature [hunk 5]
```diff
@@ -24,4 +27,15 @@ Feature: web can be navigated through urls
     And "Alice" shares the following resource using the direct url navigation
       | resource  | recipient | type | role     | resourceType |
       | lorem.txt | Brian     | user | Can view | file         |
+    When "Alice" opens the file "lorem.txt" through the URL
+    And "Alice" enters the text "Hello world" in editor "TextEditor"
+    And "Alice" saves the file viewer
+    And "Alice" closes the file viewer
```
Comment (ScharfViktor) [inline tests/e2e/cucumber/features/smoke/urlJourneys.feature:33]: if you want to check file opening and you don't need change file content -> you can use existing steps
```
Then "Alice" is in a text-editor
And "Alice" closes the file viewer
```

Comment (SwikritiT) [reply tests/e2e/cucumber/features/smoke/urlJourneys.feature:33]: Yes, I just want to check the file opening. I'll refactor the step thanks!

**Review by ScharfViktor — APPROVED** [review]
:thumbsup:

**Pull request merged**